lib/internal/Magento/Framework/Css/PreProcessor/Instruction/MagentoImport.php

```diff
@@ -57,31 +67,50 @@ class MagentoImport implements PreProcessorInterface
      */
     private $themeProvider;
 
+    /**
+     * @var DeploymentConfig
+     */
+    private $deploymentConfig;
+
+    /**
+     * @var ModuleManager
+     */
+    private $moduleManager;
+
     /**
      * @param DesignInterface $design
      * @param CollectorInterface $fileSource
      * @param ErrorHandlerInterface $errorHandler
-     * @param \Magento\Framework\View\Asset\Repository $assetRepo
-     * @param \Magento\Framework\View\Design\Theme\ListInterface $themeList
+     * @param AssetRepository $assetRepo
+     * @param ThemeListInterface $themeList
+     * @param DeploymentConfig|null $deploymentConfig
+     * @param ModuleManager|null $moduleManager
      */
     public function __construct(
         DesignInterface $design,
         CollectorInterface $fileSource,
         ErrorHandlerInterface $errorHandler,
-        \Magento\Framework\View\Asset\Repository $assetRepo,
-        \Magento\Framework\View\Design\Theme\ListInterface $themeList
+        AssetRepository $assetRepo,
+        ThemeListInterface $themeList,
+        ?DeploymentConfig $deploymentConfig = null,
+        ?ModuleManager $moduleManager = null
     ) {
         $this->design = $design;
         $this->fileSource = $fileSource;
         $this->errorHandler = $errorHandler;
         $this->assetRepo = $assetRepo;
         $this->themeList = $themeList;
+        $this->deploymentConfig = $deploymentConfig ?? ObjectManager::getInstance() ->get(DeploymentConfig::class);
+        $this->moduleManager = $moduleManager ?? ObjectManager::getInstance()->get(ModuleManager::class);
     }
 
     /**
-     * {@inheritdoc}
+     * Transform content and/or content type for the specified preprocessing chain object
+     *
+     * @param Chain $chain
+     * @return void
      */
-    public function process(\Magento\Framework\View\Asset\PreProcessor\Chain $chain)
+    public function process(Chain $chain): void
```

Comment:
Please revert adding return type for public or protected methods, that's backward incompatible change

Reply:
```suggestion
    public function process(Chain $chain)
```